### DISCUSSION TITLE
编译结果来看，导师信息居中存在问题

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#165] [Q&A] [answered]
使用博士论文模板 1.1.0版本，
macOS系统，
本地编译，
从截图的红色边框可以看出，导师信息明显偏左 （遮挡信息时刻意保留了部分笔画以更好展示结果）

<img width="538" alt="截屏2022-10-31 16 33 58" src="https://user-images.githubusercontent.com/30745407/198966191-2169cc67-002a-4011-a721-e86915687fdb.png">

**Comment by atxy-blip** [comment, marked as answer]
已解决，见 <https://github.com/nju-lug/NJUThesis/commit/d5089e6095a786f58b0804c158cc1df11dfa3805>。

这条按说也应该投到 issue 区……

**Reply by dreistein543** [reply]
感谢修复！
不放在issue区主要是因为issue的提交需要提供太多信息，不像discussion里那么方便 TAT

**Reply by atxy-blip** [reply]
可以使用 New issue 页面下方的 [Open a blank issue](https://github.com/nju-lug/NJUThesis/issues/new) 跳过模板，然而模板的本意就是使问题报告规范化（
